frontend_tests/node_tests/hotkey.js

```diff
@@ -402,11 +402,6 @@ run_test('motion_keys', () => {
     assert_mapping('down_arrow', 'subs.switch_rows');
     overlays.streams_open = return_false;
 
-    overlays.lightbox_open = return_true;
-    assert_mapping('left_arrow', 'lightbox.prev');
-    assert_mapping('right_arrow', 'lightbox.next');
-    overlays.lightbox_open = return_false;
```

Comment:
I don't think deleting these tests is a good idea.

Reply:
I did not get how I can test for two behaviors from the same key. Can you please help me with making the tests right.